cmd/tsbs_generate_data/iot/diagnostics.go

```diff
@@ -24,16 +24,31 @@ var (
 
 	diagnosticsFields = []common.LabeledDistributionMaker{
 		{
-			Label:             labelFuelState,
-			DistributionMaker: func() common.Distribution { return &customFuelDistribution{common.CWD(fuelUD, 0, maxFuel, maxFuel)} },
+			Label: labelFuelState,
+			DistributionMaker: func() common.Distribution {
+				return common.FP(
+					&customFuelDistribution{common.CWD(fuelUD, 0, maxFuel, maxFuel)},
```

Comment:
when i last talked with @davidkohn88 he mentioned that this seemed to jump around a bit more erratically than one might expect. Does that seem to be a problem? It is a clamped random walk so I would imagine it wouldn't but haven't seen the output.

Reply:
I believe he was talking about current load which should be modeled differently (staying at one state for periods of time and changing linearly from state to state to simulate loading/unloading) but that would require some thought how to do it properly. 

Do you have any suggestions there?

Reply:
I'll have to think about it a bit, but it probably requires a different distributor. Maybe something like a ClampedRandomWalk that also takes a parameter for % chance that the value stays the same which we can set to like 80%.

Reply:
Can be a separate PR, let's merge this in for now.